lints/lint_dnsname_wildcard_left_of_public_suffix.go

```diff
@@ -0,0 +1,54 @@
+package lints
+
+import (
+	"github.com/weppos/publicsuffix-go/publicsuffix"
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/util"
+)
+
+type DNSNameWildcardLeftofPublicSuffix struct {
+	// Internal data here
+}
+
+func (l *DNSNameWildcardLeftofPublicSuffix) Initialize() error {
+	return nil
+}
+
+func (l *DNSNameWildcardLeftofPublicSuffix) CheckApplies(c *x509.Certificate) bool {
+	return util.IsSubscriberCert(c)
+}
+
+func wildcardLeftOfPublicSuffix(domain string) (bool, ResultStruct) {
+	parsedDomain, err := publicsuffix.Parse(domain)
+	if err != nil {
+		return true, ResultStruct{Result: NA}
```

Comment:
Why is this NA and not Fatal? If we can't parse the domain of a certificate, that seems like a problem worth reporting.

Reply:
Hrm. My thought was that if it's _not_ a domain, then this test shouldn't apply. So, a CA that throws random crap into the SANs or CN. But I can make this a fatal if you think it makes more sense.